build.sbt

```diff
@@ -25,30 +27,29 @@ versionWithGit ++ versionSettings
 
 val sprayJsonV = "1.3.2"
 
-val lenthallV = "0.21"
+val lenthallV = "0.23-665daf8-SNAP"
 
 resolvers ++= List(
   "Broad Artifactory Releases" at "https://artifactory.broadinstitute.org/artifactory/libs-release/"
 )
 
 libraryDependencies ++= Seq(
   "org.broadinstitute" %% "lenthall" % lenthallV,
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.4.0",
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
   "io.spray" %% "spray-json" % sprayJsonV,
   /*
   Exclude test framework cats-laws and its transitive dependency scalacheck.
   If sbt detects scalacheck, it tries to run it.
   Explicitly excluding the two problematic artifacts instead of including the three (or four?).
   https://github.com/typelevel/cats/tree/v0.7.2#getting-started
    */
-  "org.typelevel" %% "cats" % "0.7.2"
+  "org.typelevel" %% "cats" % "0.9.0"
     exclude("org.typelevel", "cats-laws_2.11")
```

Comment:
@kshakir I just noticed this (would exist on centaur as well) - should I throw in a 2.12 line for these as well?